samples/customer-registry-quickstart/src/main/proto/customer/customer_view.proto

```diff
@@ -32,6 +38,8 @@ service CustomerByName {
     };
     option (akkaserverless.method).view.update = {
       table: "customers"
+      // TODO in codegen: added this to trigger codegen
+      transform_updates: true
```

Comment:
```suggestion
```

This wasn't necessary after all. It got removed in https://github.com/lightbend/akkaserverless-java-sdk/pull/216/files